acasclient/lsthing.py

```diff
@@ -2100,12 +2101,14 @@ class SimpleLink(BaseModel):
     """
     _fields = ['verb', 'subject', 'object', 'metadata', 'results']
 
-    def __init__(self, verb=None, subject=None, object=None, metadata={}, results={}, recorded_by=None, itx_ls_thing_ls_thing=None,
+    def __init__(self, verb=None, subject=None, object=None, metadata=None, results=None, recorded_by=None, itx_ls_thing_ls_thing=None,
                  subject_type=None, object_type=None):
         """
         Create a link of form: "{subject} {verb} {object}" where {subject} and {object} are instances of SimpleLsThing
         examples: "{batch} {instantiates} {parent}", "{literature reference} {contains} {pdb structure}"
         """
+        metadata = metadata or {}
```

Comment:
Similar question here - is there a reason this is at the top and not on line 2153?

Reply:
Same as above